client/components/site-selector/index.jsx

```diff
@@ -506,9 +511,9 @@ const navigateToSite =
 			}
 		}
 
-		function getSiteBasePath( site ) {
+		function getSiteBasePath( siteObj ) {
```

Comment:
What is the reason behind this change? I think `site` is a common name for the object that contains information about a site. When we refer to a specific site's piece of data, we normally mention that in the variable's name, for instance, `siteSlug` or `siteId`.

Reply:
This change was made because we move the site to the parent scope. I realised this change is not required and we don't have to pass site since it is already declared in the parent scope - here is the change - https://github.com/Automattic/wp-calypso/pull/62857/commits/8d90b8e56e63ec5efab1cdc79b7c77ff83439b6d